app/src/org/commcare/tasks/MultimediaInflaterTask.java

```diff
@@ -14,79 +13,10 @@
 import java.util.zip.ZipEntry;
 import java.util.zip.ZipFile;
 
-public abstract class MultimediaInflaterTask<R> extends CommCareTask<String, String, Boolean, R> {
+public abstract class MultimediaInflaterTask<R> extends UnzipTask<R> {
 
     @Override
-    protected Boolean doTaskBackground(String... params) {
```

Comment:
MultimediaInflaterTask was completely same as `UnzipTask`, so exteding from `UnzipTask` instead and removing the duplicate code

Reply:
nice